web/src/App.vue

```diff
@@ -20,7 +20,7 @@ export default {
           text: 'VIIME',
           to: { name: 'Root' },
         },
-        ...this.$route.matched.map((route) => {
+        ...this.$route.matched.filter(route => !route.meta.hidden).map((route) => {
```

Comment:
For routes without the `meta` property, won't you get `cannot read property 'hidden' of undefined` ?

That case should probably be handled since `meta` isn't a sanctioned router property.

Reply:
what do you mean, meta is a valid field: https://router.vuejs.org/guide/advanced/meta.html

Reply:
Oh, weird.  I've never noticed it.